frontend/src/components/eMIB/Inbox.jsx

```diff
@@ -24,51 +22,62 @@ const styles = {
   }
 };
 
+function initializeFalseArray(length) {
+  var arr = [];
+  for (var i = 0; i < length; i++) {
+    arr.push(false);
+  }
+  return arr;
+}
+
 class Inbox extends Component {
   static propTypes = {
-    inboxSpecs: PropTypes.array.isRequired
+    inboxLength: PropTypes.number.isRequired
   };
 
   state = {
-    currentEmail: 0
+    currentEmail: 0,
+    emailRead: initializeFalseArray(this.props.inboxLength),
+    emailReplies: initializeFalseArray(this.props.inboxLength)
   };
 
   changeEmail = index => {
-    this.setState({ currentEmail: index - 1 });
+    var array = this.state.emailRead;
+    array[index] = true;
+    array[this.state.currentEmail] = true; // previous email is marked as opened
+    this.setState({ currentEmail: index, emailRead: array });
   };
 
   render() {
+    const INBOX_SPECS = getInboxContent();
```

Comment:
typically consts inside of a component are `camelCase`

Reply:
Fixed in f8a0a0b